dangerfile.js

```diff
@@ -25,12 +25,26 @@ import { danger, warn, message } from 'danger';
 
 const fileChanges = [...danger.git.modified_files, ...danger.git.created_files];
 
+// Always be nice.
+message('Thanks for the PR 🎉.');
+
 // If any of the packages have changed, the changelog should have been updated.
 const changelogModified = includes(fileChanges, 'changelog.md');
-const packagesModified = fileChanges.filter(filePath => filePath.startsWith('packages/')).length > 0;
+const packagesModified = fileChanges.filter(filePath => (
+  filePath.startsWith('packages/') || filePath.startsWith('native/packages/')
+)).length > 0;
 if (packagesModified && !changelogModified) {
   warn('One or more packages have changed, but `changelog.md` wasn\'t updated.');
 }
 
-// Always be nice
-message('Thanks for the PR 🎉.');
+// If source files have changed, the snapshots should have been updated.
+const sourceFilesModified = fileChanges.filter(filePath => (
+  // packages/(one or more chars)/src/(one or more chars).js
+  filePath.match(/packages\/.+\/src\/.+\.js/) && !includes(filePath, '-test.')
```

Comment:
What if the changed file(s) are in `/native/packages/...`??

Reply:
Ignore me - it would still be a match!!

Reply:
It still works, because the regex doesn't have the start-of-line thing and will therefore match any part of the path.